packages/next-swc/crates/napi/src/transform.rs

```diff
@@ -73,7 +73,7 @@ impl Task for TransformTask {
                 try_with_handler(
                     self.c.cm.clone(),
                     swc_core::base::HandlerOpts {
-                        color: ColorConfig::Never,
+                        color: ColorConfig::Always,
```

Comment:
Is it possible to ship `v0.43.38` and color support in a separate PR? So that the swc minifier issue can be fixed in a canary release, while color support can be investigated and discussed.